runtime/drivers/clickhouse/olap.go

```diff
@@ -76,6 +76,11 @@ func (c *connection) Exec(ctx context.Context, stmt *drivers.Statement) error {
 }
 
 func (c *connection) Execute(ctx context.Context, stmt *drivers.Statement) (res *drivers.Result, outErr error) {
+	// Log query if enabled (usually disabled)
+	if c.config.LogQueries {
```

Comment:
Also needs this change in `Exec` method. Unlike duckDB and druid, clickhouse's `Exec` method does not internally call `Execute` because the underlying driver errors out if running `Query` method for a SQL that does not return any rows.

Reply:
Good catch, fixed!